nrpy/infrastructures/CarpetX/general_relativity/rhs_eval.py

```diff
@@ -0,0 +1,270 @@
+"""
+Generates function to compute the right-hand sides of the BSSN evolution equations.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+import re
+from collections import OrderedDict as ODict
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+import sympy
```

Comment:
please use `import sympy as sp`, for consistency across codebase.

Reply:
Change applied